tsql/TSqlParser.g4

```diff
@@ -1894,7 +1894,7 @@ func_body_returns_select
   :RETURNS TABLE
   (WITH function_option (',' function_option)*)?
   AS?
-  RETURN select_statement
+  RETURN '('? select_statement ')'?
```

Comment:
Incorrect. The proper code:
```
RETURN ('(' select_statement ')' | select_statement)
```

Reply:
Thank you Ivan. How about two other places in the grammar file that have '('? ? Would your suggestion apply to them as well to enforce balanced parentheses?

`receive_statement
    : '('? RECEIVE (ALL | DISTINCT | top_clause | '*')
      (LOCAL_ID '=' expression ','?)* FROM full_table_name
      (INTO table_variable=id (WHERE where=search_condition))? ')'?
    ;`

`create_or_alter_procedure
    : ((CREATE (OR ALTER)?) | ALTER) proc=(PROC | PROCEDURE) func_proc_name (';' DECIMAL)?
      ('('? procedure_param (',' procedure_param)* ')'?)?
      (WITH procedure_option (',' procedure_option)*)?
      (FOR REPLICATION)? AS sql_clauses
    ;`

Reply:
They also should be fixed.